docs/provisioning_guide/src/asciidoc/_chapters/introduction.adoc

```diff
@@ -0,0 +1,466 @@
+standard JDBC and ODBC APIs.
+
+You may choose whether to add Trafodion to an existing Hadoop environment or to create a standalone
+Hadoop environment specifically for Hadoop.
+
+This guide assumes that a Hadoop environment exists upon which your provisioning Trafodion. Refer to
+<<requirements-hadoop-software,Hadoop Software>> for information about what Hadoop software is required
+Trafodion.
+
+[[introduction-security-considerations]]
+== Security Considerations
+
+The following users need be considered for Trafodion:
+
+* *Provisioning User*: A Linux-level user that performs the Trafodion provisioning tasks. This user ID
+requires `sudo` access and passwordless ssh among the nodes where Trafodion is installed. In addition,
+this user ID requires access to Hadoop distribution, HDFS, and HBase administrative users to change
+respective environment's configuration settings per Trafodion requirements. Refer to
+<<requirements-trafodion-provisioning-user,Trafodion Provisioning User>> for more information
+about the requirements and usage associated with this user ID.
+
+* *Runtime User*: A linux-level user under which the Trafodion software runs. This user ID must be registered
+as a user in the Hadoop Distributed File System (HDFS) to store and  access objects in HDFS, HBase, and Hive.
+In addition, this  user ID requires passwordless access among the nodes where Trafodion is installed.
+Refer to <<requirements-trafodion-runtime-user,Trafodion Runtime User>> for more information about this user ID.
+
+* *Trafodion Database Users*: Trafodion users are managed by the Trafodion security features (grant, revoke, etc.),
+which can be integrated with LDAP if so desired. These users are referred to as *database users* and
+do not have direct access to the operating system. Refer to 
+http://trafodion.apache.org/docs/sql_reference/index.html#register_user_statement[Register User],
+http://trafodion.apache.org/docs/sql_reference/index.html#grant_statement[Grant], and other SQL statements
+in the http://trafodion.apache.org/docs/sql_reference/index.html[Trafodion SQL Reference Manual] for
+more information about managing Trafodion Database Users.
+ +
+ +
+Optionally, you can enable Trafodion Security. If you do not enable security in Trafodion, then a client interface
+to Trafodion may request a user name and password, but Trafodion ignores the user name and password entered in the
+client interface, and the session runs as the database *root* user, `DB__ROOT`, without restrictions. If you want
+to restrict users, restrict access to certain users only, or restrict access to an object or operation, then you must
+enable security, which enforces authentication and authorization. Refer to
+<<enable-security,Enable Security>> for more information about this option.
+
+[[introduction-provisioning-options]]
+== Provisioning Options
+
+Trafodion ships with a set of scripts (the Trafodion Installer) that takes care of many of the installation and upgrade
```

Comment:
My comment is incorrect; "takes care" is correct. ("Takes care" refers to "set")